version.js

```diff
@@ -151,16 +151,30 @@ function tryParseVersionBranch(branchName) {
 
 /**
  * Checks the PR to see if the major version should be incremented based on labels.
+ * @param {{ owner: string, repo: string }} repo
  * @param {string} commitMessage
- * @returns {Promise<'minor' | 'major'>}
+ * @returns {Promise<SemVer>}
  */
-async function getIncrementType(commitMessage) {
+async function getDefaultBranchNextVersion(repo, commitMessage) {
```

Comment:
I would like to see some inline comments here; I'm not sure I understand the context for each situation here.